frontend/src/components/Ambient/AmbientBadge.tsx

```diff
@@ -2,13 +2,14 @@ import * as React from 'react';
 import { Label, Tooltip, TooltipPosition } from '@patternfly/react-core';
 
 type AmbientLabelProps = {
-  tooltip: boolean;
+  message?: string;
   style?: React.CSSProperties;
+  tooltip: boolean;
```

Comment:
This is minor but sort of odd.   I think you should just combine `tooltip` and `message` into just `tooltip?` and always pass in the tooltip content from the parent.

Reply:
Sure, added the change to use just `tooltip`.